runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStyle.java

```diff
@@ -27,7 +27,7 @@
 import lombok.Getter;
 import net.runelite.api.Skill;
 
-enum AttackStyle
+public enum AttackStyle
```

Comment:
This isn't the right way to go about this. If we want to make this code available to multiple plugins, it should be moved to a non-plugin package.

Reply:
Sorry, I'm not familiar enough to know where this could or should be moved. If you could suggest a place, I'm happy to move it.

For reference, I thought this was acceptable because these are also public and cross-plugin:

- `XpActionType`
- `XpGoalTimeType`
- `XpPanelLabel`
- `XpProgressBarLabel`